Dockerfile

```diff
@@ -4,9 +4,9 @@ ENV DOWNLOAD_URL https://download.limesurvey.org/latest-master/limesurvey6.4.0+2
 ENV DOWNLOAD_SHA256 2631c8c03d96e2e5ba3f8c80f9e8488825c3259d6a5b8c1148400d3de4419382
 
 # install the PHP extensions we need
-RUN apt-get update && apt-get install -y unzip libc-client-dev libfreetype6-dev libmcrypt-dev libpng-dev libjpeg-dev libldap-common libldap2-dev zlib1g-dev libkrb5-dev libtidy-dev libzip-dev libsodium-dev && rm -rf /var/lib/apt/lists/* \
+RUN apt-get update && apt-get install -y unzip libc-client-dev libfreetype6-dev libmcrypt-dev libpng-dev libjpeg-dev libldap-common libldap2-dev zlib1g-dev libkrb5-dev libtidy-dev libzip-dev libsodium-dev libpq-dev libonig-dev libzip-dev && rm -rf /var/lib/apt/lists/* \
```

Comment:
minor change her libzip-dev is specified twice

Reply:
Thanks, fixed.